src/main/java/retryAnalyzer/Retry.java

```diff
@@ -0,0 +1,28 @@
+package retryAnalyzer;
+
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestResult;
+
+/***
+ * RetryAnalyzer helps to check if the failure was caused by a product bug or another reason.
```

Comment:
In case of a bug test should not be rerun

Reply:
change the java doc